### DISCUSSION TITLE
Profiles Managing

### DISCUSSION THREAD

**Discussion by SoloPringle** [runelite/runelite#16305] [Ideas]
Hi,
I was messing with the new Profiles features and thought there should be someway to reorder your profiles in the list.
As of right now, they only appear in the order you've created them. 
A drag and drop feature or up and down arrows to allow reordering would be useful for a lot of users, I'm sure.

I don't know if this was already going to be implemented later on or not but thought I'd share.

**Comment by Nightfirecat** [comment]
This was added in 182b0d769355c8276412a4ef44966f5618b57edb
